src/main/java/frc/robot/swerve/FalconDriveController.java

```diff
@@ -20,7 +21,11 @@ public class FalconDriveController implements DriveController {
     public FalconDriveController(int id) {
         m_motor = new TalonFX(id);
         m_motor.setInverted(MOTOR_INVERTED);
+        m_motorCurrentConfigs = new CurrentLimitsConfigs();
+        m_motorCurrentConfigs.withStatorCurrentLimit(CURRENT_LIMIT);
```

Comment:
I don't think we want to limit the stator current. We want the supply current limits.